bin/einhorn

```diff
@@ -53,7 +53,7 @@ arguments:
 
 Each address is specified as an ip/port pair, possibly accompanied by options:
 
-    ADDR := (IP:PORT)[<,OPT>...]
+    ADDR := (IP:PORT)[<,FLAG>...]
```

Comment:
Since we're editing this, let's make clear that the flags are comma-separated? I remember that tripping someone up recently